Sources/AppCoinsSDK/Domain/Repositories/Currency/CurrencyRepository.swift

```diff
@@ -17,11 +17,11 @@ internal class CurrencyRepository: CurrencyRepositoryProtocol {
         if let userCurrency = self.UserCurrency.getValue(forKey: "userCurrency") {
             completion(.success(userCurrency))
         } else {
-            AppCoinBillingService.convertCurrency(money: "1.0", fromCurrency: Coin.APPC, toCurrency: nil) { result in
+            AppCoinBillingService.convertCurrency(money: "1.0", fromCurrency: "APPC", toCurrency: nil) { result in
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done